app/res/layout/component_entity_detail_item.xml

```diff
@@ -3,19 +3,23 @@
     xmlns:customViews="http://schemas.android.com/apk/lib/org.commcare.android.view"
     android:layout_width="fill_parent"
     android:layout_height="wrap_content"
-    android:background="@drawable/grey_bordered_box"
-    android:orientation="horizontal" android:weightSum="10">
+    android:orientation="horizontal" android:weightSum="10"
+    android:paddingLeft="8dp"
+    android:paddingRight="8dp"
+    >
+    <!--android:background="@drawable/grey_bordered_box"-->
 
     <TextView
         android:id="@+id/detail_type_text"
         style="@style/EntityItemTextView"
         android:layout_width="0dp"
         android:layout_height="fill_parent"
-        android:background="@drawable/blue_tabbed_box"
-        android:gravity="top|left|center_vertical"
+        android:gravity="left|center_vertical"
         android:text="Text"
-        android:textAppearance="@style/EntityItemText"
+        android:textStyle="bold"
         android:typeface="normal" android:layout_weight="4" android:visibility="visible"/>
+    <!--android:background="@drawable/blue_tabbed_box"-->
+    <!--android:textAppearance="@style/EntityItemText"-->

```

Comment:
Is there a reason we didn't just update this style? It's still used somewhere else? 

Would be good to use a different one instead of custom